src/orchestrator/tools.py

```
@@ -59,7 +59,19 @@ async def requirements(
             config_with_recursion,
         )
 
-        return result["summary"]
+        return Command(
```

Ruff (0.11.9)

Undefined name `Command`

(F821)